week1/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.8-slim-buster
+RUN useradd --create-home appuser
+USER appuser
+
+# set a directory for the app
+WORKDIR /usr/src/app
+
+# copy all the files to the container
+COPY flask-app .
+
+# install dependencies
+RUN pip install --no-cache-dir -r requirements.txt --user
```

Comment:
let's install deps before 

```
# copy all the files to the container
COPY flask-app .
```

so we can really on cache

Reply:
I thought we first need to copy the requirements file into docker and then install the deps.
I will check if it's possible to swap them

Reply:
<img width="797" alt="image" src="https://user-images.githubusercontent.com/33032498/190843029-0c767ec7-1573-4673-a53d-448ac5b94cc9.png">

Reply:
@truskovskiyk

Reply:
Or did I not understand something?

Reply:
I just removed --no-cache-dir flag, now everything is cached